include/exadg/grid/grid.cpp

```diff
@@ -233,15 +246,22 @@ Grid<dim>::do_create_triangulation(
 
     unsigned int const group_size = 1;
 
+    typename dealii::TriangulationDescription::Settings triangulation_description_setting;
+
+    if(data.create_coarse_triangulations)
+      triangulation_description_setting = dealii::TriangulationDescription::default_setting;
+    else
+      triangulation_description_setting =
+        dealii::TriangulationDescription::construct_multigrid_hierarchy;
+
     // TODO SIMPLEX: this will not work in case of simplex meshes
     auto const description = dealii::TriangulationDescription::Utilities::
-      create_description_from_triangulation_in_groups<dim, dim>(
-        serial_grid_generator,
-        serial_grid_partitioner,
-        triangulation->get_communicator(),
-        group_size,
-        dealii::Triangulation<dim>::none,
-        dealii::TriangulationDescription::construct_multigrid_hierarchy);
+      create_description_from_triangulation_in_groups<dim, dim>(serial_grid_generator,
+                                                                serial_grid_partitioner,
+                                                                triangulation->get_communicator(),
+                                                                group_size,
+                                                                mesh_smoothing,
```

Comment:
```suggestion
                                                                triangulation->get_mesh_smoothing(),
```

Reply:
Thanks! This works, but we rely on `triangulation_description_setting` changing the mesh smoothing from none to limit_level_difference_at_vertices in case of local smoothing.

Reply:
Why is it `none`? I don't understand this.

Reply:
Because if I do `triangulation->get_mesh_smoothing()`, I get the mesh smoothing of the triangulation built in:
https://github.com/exadg/exadg/blob/22aebf621ea018865c6838eda526dd15b6a3e8bb/include/exadg/grid/grid.cpp#L90-L91
which is `none`. The following code sniplet

```
    std::cout << "Mesh Smoothing before create_description_from_triangulation_in_groups"
              << std::endl;
    switch(triangulation->get_mesh_smoothing())
    {
      case dealii::Triangulation<dim>::none:
        std::cout << "  none" << std::endl;
        break;
      case dealii::Triangulation<dim>::limit_level_difference_at_vertices:
        std::cout << "  limit_level_difference_at_vertices" << std::endl;
        break;
    }

    // TODO SIMPLEX: this will not work in case of simplex meshes
    auto const description = dealii::TriangulationDescription::Utilities::
      create_description_from_triangulation_in_groups<dim, dim>(
        serial_grid_generator,
        serial_grid_partitioner,
        triangulation->get_communicator(),
        group_size,
        triangulation->get_mesh_smoothing(),
        dealii::TriangulationDescription::Settings::construct_multigrid_hierarchy);

    triangulation->create_triangulation(description);

    std::cout << "Mesh Smoothing after create_description_from_triangulation_in_groups"
              << std::endl;
    switch(triangulation->get_mesh_smoothing())
    {
      case dealii::Triangulation<dim>::none:
        std::cout << "  none" << std::endl;
        break;
      case dealii::Triangulation<dim>::limit_level_difference_at_vertices:
        std::cout << "  limit_level_difference_at_vertices" << std::endl;
        break;
    }
```
gives this output:
```
Mesh Smoothing before create_description_from_triangulation_in_groups
  none
Mesh Smoothing after create_description_from_triangulation_in_groups
  limit_level_difference_at_vertices
```